.github/workflows/test_common.yml

```diff
@@ -47,5 +47,5 @@ jobs:
       run: |
         # Setting HUGGINGFACE_CO_STAGING to true for only one job of the matrix as the staging tests cannot run in parallel.
         export HUGGINGFACE_CO_STAGING=${{ matrix.python-version == '3.8' && matrix.os == 'ubuntu-20.04' }}
-        pytest tests/test_*.py
+        pytest tests/test_examples.py tests/test_utils.py
```

Comment:
No I think we should test anything there, hence "test common".
Since export is a feature on its own, let's make a directory `tests/export` and put `test_export.py` there (and all the futur tests related to export there as well)